ims.yaml

```diff
@@ -0,0 +1,672 @@
+            items:
+              $ref: '#/definitions/NameResource'
+          address:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/AddressResource'
+          contactInfo:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/ContactInfoResource'
+          demographics:
+            $ref: '#/definitions/DemographicsResource'
+          role:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/RoleResource'
+          agent:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/AgentResource'
+          credentials:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/CredentialsResource'
+        required:
+          - status
+          - dateLastModified
+          - formattedName
+          - name
+        additionalProperties: false
+  PersonResult:
+    description: ...tbd (Class description)...
+    type: object
+    properties:
+      data:
+        $ref: '#/definitions/PersonResource'
+    additionalProperties: false
+  PersonSetResult:
+    description: ...tbd (Class description)...
+    type: object
+    properties:
+      data:
+        type: array
+        minItems: 0
+        items:
+          $ref: '#/definitions/PersonResource'
+    additionalProperties: false
+  RoleResource:
+    description: ...tbd (Class description)...
+    type: object
+    properties:
+      roleType:
+        description: ...tbd (Enumeration description)...
+        type: string
+        enum:
+          - PRIMARY
+          - SECONDARY
+      role:
+        description: ...tbd (Enumeration description)...
+        type: string
+        enum:
+          - ADMINISTRATOR
+          - AIDE
+          - GUARDIAN
+          - PARENT
+          - ...tbc...
+      organization:
+        $ref: '#/definitions/GUIDRefResource'
+    required:
+      - roleType
+      - role
+      - organization
+    additionalProperties: false
+  imsx_CodeMinorResource:
+    description: This is the container for the set of code minor status codes reported in the responses from the Service Provider.
+    type: object
+    properties:
+      imsx_codeMinorField:
+        description: Each reported code minor status code.
+        type: array
+        minItems: 1
+        items:
+          $ref: '#/definitions/imsx_CodeMinorFieldResource'
+    required:
+      - imsx_codeMinorField
+    additionalProperties: false
+  imsx_CodeMinorFieldResource:
```

Comment:
Since these two `imsx*` definitions aren't being used anywhere, should they be deleted?